container/trie/sparse_merkle.go

```diff
@@ -214,18 +213,15 @@ func VerifyMerkleProofWithDepth(root, item []byte, merkleIndex uint64, proof [][
 	if uint64(len(proof)) != depth+1 {
 		return false
 	}
-	if depth >= 64 {
-		return false // PowerOf2 would overflow.
-	}
```

Comment:
Why?

Reply:
Because the reason for that test disappeared, there's no possible overflow now